app/helpers/collections_helper.rb

```diff
@@ -34,4 +34,10 @@ def render_collection_download_xml_link(download_format, download_link_title, co
     link_to(download_link_title, download_collection_xml_path(concept_id, download_format, revision_id),
             id: download_format)
   end
+
+  def nrt_badge(metadata)
+    if metadata['CollectionDataType'] == 'NEAR_REAL_TIME'
+      content_tag :span, 'NRT', class: 'eui-badge nrt'
```

Comment:
I know parentheses aren't required here, but if you look at where we use `content_tag` in the rest of the project, we wrap the arguments in parentheses, so it we should be consistent with that.